### PR TITLE
Introduce cpe2pkg module

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
@tisnik another helper module, not used by the rest of the project yet. PTAL :)

**Comment by msrb** [comment]
[test]

**Comment by msrb** [comment]
[test]

File: cvejob/cpe2pkg.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    def __lt__(self, other):
+        if not isinstance(other, self.__class__):
+            return False
+        return self.score < other.score
+
+    def __le__(self, other):
+        if not isinstance(other, self.__class__):
+            return False
+        return self.score <= other.score
+
```
Comment (tisnik) [inline cvejob/cpe2pkg.py]: it means that PackageNameCandidate() > valueOfDifferentType everytime?

**Comment by tisnik** [comment]
[test]

**Comment by msrb** [comment]
[test]

**Comment by msrb** [comment]
[test]

**Comment by msrb** [comment]
[test]

**Pull request merged**